lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskServiceImpl.java

```diff
@@ -1205,6 +1214,11 @@ private Task claim(String taskId, boolean forceClaim)
       throws TaskNotFoundException, InvalidStateException, InvalidOwnerException,
           NotAuthorizedException {
     String userId = taskanaEngine.getEngine().getCurrentUserContext().getUserid();
+    User user = userMapper.findById(userId);
+    String userLongName = null;
+    if (user != null) {
+      userLongName = userMapper.findById(userId).getLongName();
```

Comment:
why are you making another call to the database here? You already have the user entity from line 1217 to retrieve the long name from, right?

Why is the logic here different than the one above from line 1110-1116? :)

Reply:
The additional database call was on accident.  I will fix it :)

The logic is different here, because claimActionsOnTask is not inside a lambda expression. This way, userLongName doesn't have to be final.